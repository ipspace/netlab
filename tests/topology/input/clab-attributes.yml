#
# clab specific provider attributes, see netsim/providers/clab.yml
#
---
provider: clab

nodes:
  n:
    device: frr
    clab:
      binds: []
      kind: frr
      config_templates: []
      type: unknown
      cmd: '/bin/bash'
      env:
        FOO: "BAR"
      ports: []
      image: quay.io/frrouting/frr:9.1.0
      startup-config: 'start.cfg'
      srl-agents: []
      license: 'license.txt'
      runtime: 'docker'
