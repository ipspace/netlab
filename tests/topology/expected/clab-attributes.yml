input:
- topology/input/clab-attributes.yml
- package:topology-defaults.yml
name: input
nodes:
  n:
    af:
      ipv4: true
    box: ghcr.io/nokia/srlinux:24.3.2
    clab:
      binds: []
      cmd: /bin/bash
      config_templates: []
      env:
        FOO: BAR
      image: srl/linux:latest
      kind: srlinux
      license: license.txt
      ports: []
      runtime: docker
      srl-agents: []
      startup-config: start.cfg
      type: ixrd2l
    device: srlinux
    hostname: clab-input-n
    id: 1
    interfaces: []
    loopback:
      ifindex: 0
      ifname: lo0.0
      ipv4: 10.0.0.1/32
      neighbors: []
      type: loopback
      virtual_interface: true
    mgmt:
      ifname: mgmt0
      ipv4: 192.168.121.101
      mac: 08:4f:a9:00:00:01
    name: n
provider: clab
